examples/client-logger/app-insights-logger/src/test/components/App.test.tsx

```diff
@@ -7,8 +7,9 @@ import { render, screen } from "@testing-library/react";
 import React from "react";
 import { App } from "../../components";
 
+// TODO: update ESM configuration and re-enable test
 describe("App Insights Example App UI test", () => {
-	it("App renders", async (): Promise<void> => {
+	xit("App renders", async (): Promise<void> => {
```

Comment:
I don't think I've seen `xit` before. We appear to use `it.skip`.
And there should be a comment directly above.